router-sharding/05_router-default.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: operator.openshift.io/v1
+kind: IngressController
+metadata:
+  finalizers:
+  - ingresscontroller.operator.openshift.io/finalizer-ingresscontroller
+  name: default
+  namespace: openshift-ingress-operator
+spec:
+  replicas: 2
+  routeSelector:
```

Comment:
Lets remove the selectors for the default ingress controller so that we don't affect the default route behavior of the cluster.